src/components/search/outsideAlerter.js

```diff
@@ -0,0 +1,18 @@
+import { useRef, useState, useEffect } from "react";
+
+export const useOutsideAlerter = (initialValue : boolean) => {
```

Comment:
I don't understand the naming convention of this hook and what it's goal is. Might be worth noodleing on the name. Also, please add JSDoc strings for custom hooks so that they show up in tool tips and give a good explanation as to the the use of the hook. [Example](https://github.com/Lambda-School-Labs/frontend-vbb-portal/blob/main/src/redux/Login.redux/Login.actions.js#L39-L45) It might feel like a bit of over kill but in 3 months you'll be grateful :)

Reply:
This was a work in-progress I failed to remove, thanks for pointing it out. It has been removed from the requested merge.